client/blocks/stats-navigation/index.js

```diff
@@ -19,8 +20,10 @@ import {
 	isSiteStore,
 } from 'state/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
 import { navItems, intervals as intervalConstants } from './constants';
 import config from 'config';
+import { isWpComFreePlan } from 'lib/plans';
```

Comment:
I noticed another day a bunch of similar ones here: https://github.com/Automattic/wp-calypso/blob/master/client/lib/products-values/index.js

...but suppose this is still the way to check this. :-)